src/NServiceBus.Transport.RabbitMQ/Routing/DirectRoutingTopology.cs

```diff
@@ -45,12 +45,16 @@ public void RawSendInCaseOfFailure(IModel channel, string address, ReadOnlyMemor
 
         public void Initialize(IModel channel, IEnumerable<string> receivingAddresses, IEnumerable<string> sendingAddresses)
         {
-            Dictionary<string, object> arguments = null;
+            Dictionary<string, object> arguments;
 
             if (queueType == QueueType.Quorum)
             {
                 arguments = new Dictionary<string, object> { { "x-queue-type", "quorum" } };
             }
+            else
```

Comment:
Newing up an empty dictionary seems stupid, but this was again due to null reference types.